requirements-optional.txt

```diff
@@ -1 +1,2 @@
 Mako==1.0.3
+raven
```

Comment:
I'm not sure if we can set a range, maybe a minimum version would be more preferable?